src/components/Modal/src/ModalLayer.vue

```diff
@@ -33,16 +40,27 @@ import {
 import modalApi from './modal-api';
 
 const apiMixin = {
+	inject: {
+		currentLayer: {
+			default: undefined,
+			from: modalApi,
+		},
+	},
+
 	provide() {
 		const vm = this;
+		// const id = Math.random().toString().slice(2, 5);
```

Comment:
Want to clean out these inline comments?

Reply:
Every time I've cleaned them out I've put them back in a day later, they serve as really good debugging info but sure, I can remove them.